### ISSUE TITLE
All  focus indicators need to not rely on colour alone.

### ISSUE THREAD

**Issue by MarkGOC** [dto-btn/ssc-assistant#582]
Some of our focus indicators are only done using color. 

Eg if you tab from chat to chat for example in the left menu. We just need to add a bounding box around the element so that focus is not only shown by colour.

A single line dark border should be fine.    Black or our blue colour.   We just want to pick a colour that is dark and works with our UI theme.  We do have a dark blue colour as well that can be used.

We just need to be consistant.

The border in our new BR feature for prompt suggestions is fine.  We can just use that if it is easy.
